### DISCUSSION TITLE
Alterar Kit TikTok Vanillew

### DISCUSSION THREAD

**Discussion by Batatoones** [leogianfagna/IssueTracker#925] [Opinioes]
### Descrição da tarefa

Kit Tiktok é bem inútil para o jogador podendo ser melhorado.

**Comment by ghost** [comment]
Sugestão Criador F1lipe 

Henesys:
![image](https://github.com/user-attachments/assets/79bb53d8-31d1-442a-aa5a-44d0b437c8ad)

Vanillew:
![image](https://github.com/user-attachments/assets/7173765c-a6b2-44f8-986c-90f461425003)
